### PR TITLE
Makes the code team own the code, as they should

### PR DESCRIPTION
**Pull request by adamsong**
Does as the title says, makes `/code` and `/yogstation/code` owned by the code team.

### CODE REVIEW

**Comment by alexkar598** [comment]
https://github.com/yogstation13/Yogstation/blob/4bca0d650797f08b3903bf938422ca4220948b31/.github/CODEOWNERS#L39-L42

This shit should be changable without intervention from coders tho. Bad PR

**Comment by adamsong** [comment]
> This shit should be changable without intervention from coders tho. Bad PR

It is, art owns those files

**Comment by Bibby0110** [comment]
> > This shit should be changable without intervention from coders tho. Bad PR
> 
> It is, art owns those files

We both own them. They're a subdirectory of /code/

**Comment by adamsong** [comment]
> We both own them. They're a subdirectory of /code/

Not how CODEOWNERS works, it doesn't care about that, just whichever is last takes precedence

**Comment by alexkar598** [comment]
My bad you're right, still, there are occurrences where the code changes are minor enough so that art should be able to merge shit despite containing code changes (icon and icon_state variables come to mind)

**Comment by adamsong** [comment]
As it is now, non art changes to ticker require approval from art. #12003 requires approval from art, but all I'm doing is modifying some regex

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**
